tools/crm_mon.c

```diff
@@ -3923,7 +3923,7 @@ static void crm_diff_update_v2(const char *event, xmlNode * msg)
             free(local_node);
 
         } else {
-            crm_err("Ignoring %s operation for %s %p, %s", op, xpath, match, name);
+            crm_debug("Ignoring %s operation for %s %p, %s", op, xpath, match, name);
```

Comment:
AFAICT, the previous blocks with strcmp() have considered all the types of xml objects that could contain any lrm_rsc_op.  We don't want it to report errors for the changes of transient attributes.

But yes, it probably makes sense the keep this message just in case. I'll improve it.